scripts/create-a-m-o/functions.js

```diff
@@ -67,6 +67,7 @@ const createFiles = (store, a, m, o, done) => () => {
   templateJson.name = `@axa-ch/${fileName}`;
```

Comment:
General question: Isn't the new description too obvious? I mean you can guess your description from its name. In my opinion the description was pretty okay.

Reply:
I noticed a lot of time, that when i searched for a component on npm, it was found with a "subtitle" of "TBD" or "TODO: Add descritpion". I thought it would be best to keep it as simple as obvious possible. A description how to use the component does not belong there, which sometimes also was the case.

In the end, we can still adapt it imho. But having something else than "todo" after scaffolding is important imho.